### ISSUE TITLE
REST response and typing mismatch?

### ISSUE THREAD

**Issue by silasbw** [tatumio/tatum-js#915]
Is there a mismatch in the SDK type for `NftTokenDetail.tokenType` and what the REST API returns?  It looks like the REST API returns a `type` field instead of `tokenType`.

`tokenType` (https://github.com/tatumio/tatum-js/blob/master/src/service/nft/nft.dto.ts#L77) vs `type` in the REST API response (e.g., `curl --location --request GET 'https://api.tatum.io/v4/data/balances?chain=ethereum&addresses=0x727EA45B2EB6abb2badD3dC7106d146E0Dc0450d' | jq '.' | less`)

**Comment by samuelsramko** [comment]
Hey @silasbw, the reason why we are using the different naming of the field is the SDK being a first-class citizen for us now. We are and will be integrating multiple different sources for data and unifying them inside the SDK. It means that the SDK DTOs might look different than the API responses, which is perfectly correct. If you choose to use the API for this integration, you are of course free to do it. It's just the SDK which has different format.

**Comment by silasbw** [comment]
We're using the SDK, so is your recommendation we do, like:

```ts
// The SDK types don't match the actual shape returned by the REST API: 
//    The REST API returns NftTokenDetail.type (not NftTokenDetail.tokenType)
// @ts-ignore
const tokenType = response.data[0].type
```

**Comment by samuelsramko** [comment]
O I see, Thats something else. @Hathoriel, @hehe100596 or @alexloiko can you guys check this out pls?

**Comment by samuelsramko** [comment]
Probably some typo during refactoring…

**Comment by Hathoriel** [comment]
Hello yes @samuelsramko I will look on it. 
Thanks for finding the issue @silasbw

**Comment by Hathoriel** [comment]
👋 Hello @silasbw 
The issue should be fixed in the version 3.1.7.
Hope this helps!
Have a nice day.

**Comment by silasbw** [comment]
Thanks @Hathoriel :raised_hands:
